lib/elixir/test/elixir/record_test.exs

```diff
@@ -123,6 +123,44 @@ defmodule RecordTest do
     assert user(record, :age) == :_
   end
 
+  Record.defrecord :defaults,
+    struct: ~D[2016-01-01],
+    map: %{},
+    tuple_zero: {},
+    tuple_one: {1},
+    tuple_two: {1, 2},
+    tuple_three: {1, 2, 3},
+    list: [1, 2, 3],
+    call: MapSet.new,
+    string: "abc",
+    binary: <<1, 2, 3>>,
+    charlist: 'abc'
+
+  test "records with literal defaults" do
+    assert defaults(defaults()) == [struct: ~D[2016-01-01],
```

Comment:
I can do this. This way of indenting is quite common in elixir, though. Starting with what is generated in `mix.exs`